src/api/models.tsx

```diff
@@ -79,12 +79,42 @@ export interface ApplicationDependency {
 
 // Pathfinder
 
+export type AssessmentStatus = "EMPTY" | "STARTED" | "COMPLETE";
+
 export interface Assessment {
   id?: number;
   applicationId: number;
-  status: "EMPTY" | "STARTED" | "COMPLETE";
+  status: AssessmentStatus;
   stakeholders?: number[];
   stakeholderGroups?: number[];
+  questionnaire: Questionnaire;
+}
+
+export interface Questionnaire {
+  categories: QuestionnaireCategory[];
+}
+
+export interface QuestionnaireCategory {
+  id: number;
+  order: number;
+  title?: string;
+  comment?: string;
+  questions: Question[];
+}
+
+export interface Question {
+  id: number;
+  order: number;
+  question: string;
+  description: string;
+  options: QuestionOption[];
+}
+
+export interface QuestionOption {
+  id: number;
+  order: number;
+  option: string;
+  checked: boolean;
```

Comment:
I'm missing here the risk associated.

Reply:
Thanks. My Last commit added the new field `risk`.

Reply:
:+1: